src/js/view/view.js

```diff
@@ -150,6 +150,23 @@ function View(_api, _model) {
         }
     };
 
+    this.checkOrientation = function(type) {
+        if (OS.android && Browser.chrome && _this.model.get('visibility') >= 0.75) {
+            const state = _this.model.get('state');
+            const isLandscape = type === 'landscape-primary' || type === 'landscape-secondary';
+
+            if (!isLandscape && state === 'paused') {
+                // Set fullscreen to false when going back to portrait while paused and return early
+                _this.model.set('fullscreen', false);
+                return;
+            }
+
+            if (state === 'playing') {
+                _this.model.set('fullscreen', isLandscape);
```

Comment:
Hmm don't you mean `api.setFullscreen(isLandscape)` ? Setting "fullscreen" in the model doesn't actually make the player go fullscreen, going fullscreen changes the model.